mmtrack/core/track/interpolation.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+
+def _interpolate_track(track, track_id, max_num_frames=20):
+    """Interpolate a track linearly to make the track more complete.
+
+    Args:
+        track (ndarray): With shape (N, 7). Each row denotes
+            (frame_id, track_id, x1, y1, x2, y2, score).
+        max_num_frames (int, optional): The maximum frames of a tracklet that
```

Comment:
It's not correct. First, the subject should be the track not the frames. Second, modify tracklet to track.